lib/node/incoming_message.ts

```diff
@@ -0,0 +1,15 @@
+import { ValueID } from "zwave-js";
+import { IncomingCommandBase } from "../incoming_message_base";
+import { NodeCommand } from "./command";
+
+export interface IncomingCommandNodeBase extends IncomingCommandBase {
+  nodeId: number;
+}
+
+export interface IncomingCommandNodeSetValue extends IncomingCommandNodeBase {
+  command: NodeCommand.setValue;
```

Comment:
I am not a fan of this. It just means we are declaring everything in other places and in the end it's unclear what is happening where.

Reply:
It's an improvement, we can always move things around later. Let's just go with it .

Reply:
I've did it only to make sure the property in `NodeResultTypes` matches the command ;) It's easy to misspell it especially when the number of commands grow.